File: 19) Check if the Sentence Is Pangram/code.cpp
#include <bits/stdc++.h>
using namespace std;

bool isPangram(string sentence) {
    unordered_set<char> letters;
    for (char c : sentence) {
        if (isalpha(c)) {
            letters.insert(tolower(c));
        }
    }
    return letters.size() == 26;
}

int main() {
    string sentence;
    cout << "Enter a sentence: ";
    getline(cin, sentence);

    bool result = isPangram(sentence);
    if (result) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}

File: 19) Check if the Sentence Is Pangram/code.py
def isPangram(sentence):
    letters = set()
    for c in sentence:
        if c.isalpha():
            letters.add(c.lower())
    return len(letters) == 26

sentence = input("Enter a sentence: ")
result = isPangram(sentence)
if result:
    print("Yes")
else:
    print("No")
